src/sidebar/components/SidebarView.js

```diff
@@ -142,7 +142,7 @@ function SidebarView({
 
   return (
     <div>
-      <h2 className="u-screen-reader-only">Annotations</h2>
+      <h2 className="sr-only">Annotations</h2>
```

Comment:
Ah! So, Tailwind provides a couple of [screen-reader utility styles](https://tailwindcss.com/docs/screen-readers). These are roughly similar to our own custom approach, and I'd prefer to use theirs, as it's maintained by a bunch of smart people, than to continue to try to manage our own. Thoughts?

Reply:
The CSS rule itself matches ours, with some additional properties, so this makes sense. I wish they had documented the rationale for the properties as well as we did though! I added a public H comment on the linked docs page.